programs/clearing_house/src/controller/orders.rs

```diff
@@ -2896,6 +2900,20 @@ pub fn fulfill_spot_order_with_match(
         false,
     )?;
 
+    let fee_pool_amount = get_token_amount(
```

Comment:
why not do a transfer in fulfill with serum?

Reply:
added